pyiron/vasp/base.py

```diff
@@ -428,6 +428,9 @@ def collect_errors(self):
                 if 'WARNING in EDDRMM: call to ZHEGV failed, returncode =' in l:
                     self.status.not_converged = True
                     break
+                elif 'ZBRENT: fatal error in bracketing' in l:
+                    self.status.not_converged = True
```

Comment:
Apart from setting the status to `not_converged`, I think we should also write some message in `pyiron.log` using `s.logger.warn`. This can tell the user what to do next if they encounter this error

Reply:
Yes, but maybe we should once more look into integrating Custodian and pymatgen because they have quite some more error handlers: 
https://materialsproject.github.io/custodian/_modules/custodian/vasp/handlers.html